aix.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const getSqlStatement = family => {
+  let sql =
+    "select NEXT_HOP from QSYS2.NETSTAT_ROUTE_INFO where ROUTE_TYPE='DFTROUTE'";
+
+  if (family === "v4") {
+    sql += " and CONNECTION_TYPE='IPV4'";
+  } else {
+    sql += " and CONNECTION_TYPE='IPV6'";
+  }
+
+  return sql;
+};
+
+const getGatewayInformationAsync = async family => {
+  return new Promise((resolve, reject) => {
+    const idbConnector = require("idb-connector");
```

Comment:
You should `try ... catch` these `require` statements and return the `"Unable to determine default gateway"` error on catch. Normally I'd say to let the error through to the consumer, but the module has been consistent in returning only this specific error.

Reply:
I wrapped the connector functions as well